src/main/resources/wa-task-initiation-employment-et_englandwales.dmn

```diff
@@ -2179,6 +2179,329 @@ else null</text>
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0m7xgry">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0h3wh96">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0vqtbxg">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0wmmxk8">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0nszytp">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0mu29l4">
+          <text>"Amend claim",
+"Consider decision afresh",
+"Contact the tribunal",
+"Order a witness to attend to give evidence",
+"Order other party",
+"Postpone a hearing",
+"Reconsider judgment",
+"Respondent not complied",
+"Restrict publicity",
+"Strike out all or part of the response",
+"Vary or revoke an order"</text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1b91ytk">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1fydq8v">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0z143w6">
+          <text></text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_0cq0obc">
+          <text>"ContactTribunalWithAnApplication"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_1qqwgca">
+          <text>LATEST_APPLICATION_TASK_NAME</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_0jr2m3q">
+          <text></text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_146n14u">
+          <text>"Application"</text>
+        </outputEntry>
+      </rule>
+      <rule id="DecisionRule_06zve82">
+        <inputEntry id="UnaryTests_08cxohy">
+          <text>"tseClaimantRepResponse"</text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_04ugsz2">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1pnur2c">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0vj7e3l">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0oivp2h">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0w0mde5">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_09so3ev">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1mfnod5">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0tnr00d">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1uh86rb">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_15w6ni9">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0w103v5">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0dkmyb2">
+          <text>"Claimant"</text>
```

Comment:
Noticed this when reviewing callbacks - the response from is set to Claimant Representative and not Claimant so this task might not trigger - https://github.com/hmcts/et-ccd-callbacks/pull/2259/files#diff-f007255afc0d633a2c2099ce178ce90cd4e16e7f2684a76a8d24177611e13700R211

Reply:
addressed comments [e95be0d](https://github.com/hmcts/et-wa-task-configuration/pull/263/commits/e95be0d9a145b0fd0fe51e0b1a9ff52b125285c6)